services/galley/test/integration/API/MLS.hs

```diff
@@ -1125,10 +1125,9 @@ testAppMessageSomeReachable = do
     let commitMocks =
           receiveCommitMockByDomain [bob1, charlie1]
             <|> welcomeMock
-    (([event], ftpCommit), _) <-
+    ([event], _) <-
       withTempMockFederator' commitMocks $ do
-        sendAndConsumeCommitFederated commit
-    liftIO $ ftpCommit @?= mempty
+        sendAndConsumeCommit commit
```

Comment:
have you tried translating this test case into `/integration`?  i'd give it a (time-boxed) try after this PR has been merged, `/integration` is really neat!  :)

Reply:
I entertained the idea, but many times I need at least three backends that federate, and support for testing that is still in the works.

This change in particular looks like simplifying an existing test, though.